bucket/ds4windows.json

```diff
@@ -15,7 +15,22 @@
     },
     "bin": "DS4Windows.exe",
     "extract_dir": "DS4Windows",
-    "post_install": "If (Test-Path \"$dir\\DS4Updater.exe\") { Remove-Item \"$dir\\DS4Updater.exe\" -Force }",
+    "pre_install": [
+        "'Actions.xml', 'Profiles.xml' | ForEach-Object {",
+        "    if (!(Test-Path \"$persist_dir\\$_\")) {",
+        "        New-Item \"$dir\\$_\" | Out-Null",
+        "    }",
+        "}",
+        "if (!(Test-Path \"$persist_dir\\Auto Profiles.xml\")) {",
+        "   Set-Content -Path \"$dir\\Auto Profiles.xml\" -Value '<?xml version=\"1.0\" encoding=\"utf-8\"?><Programs />'",
+        "}"
+    ],
+    "persist": [
+        "Profiles",
```

Comment:
This is folder right?

Reply:
Yes.

Reply:
https://github.com/lukesampson/scoop-extras/pull/3056#discussion_r338548603